options/locale/locale_en-US.ini

```diff
@@ -1795,6 +1798,16 @@ auths.oauth2_authURL = Authorize URL
 auths.oauth2_profileURL = Profile URL
 auths.oauth2_emailURL = Email URL
 auths.enable_auto_register = Enable Auto Registration
+auths.sspi_auto_create_users = Automatically create users
+auths.sspi_auto_create_users_helper = Allow SSPI auth method to automatically create new users on the go
```

Comment:
`on the go` sounds a little weird

Reply:
Fixed with commit https://github.com/go-gitea/gitea/pull/8463/commits/bcaf923656cd59a904c2b913247d918254f48736